drake/examples/Cars/simulateLCM.cpp

```diff
@@ -78,11 +89,22 @@ int main(int argc, char* argv[]) {
   // be reused
   DrakeJoint::FloatingBaseType floating_base_type = DrakeJoint::QUATERNION;
 
-
   auto rigid_body_sys = make_shared<RigidBodySystem>();
-  rigid_body_sys->addRobotFromFile(argv[1], floating_base_type);
-  auto const & tree = rigid_body_sys->getRigidBodyTree();
-  for (int i=2; i<argc; i++)
+
+  // The following variable, weld_to_frame, is only needed if the model is a
+  // URDF file. It is needed since URDF does not specify the location and
+  // orientation of the car's root node in the world. If the model is an SDF,
+  // weld_to_frame is ignored by the parser.
```

Comment:
Two thoughts:
- I would think the base pose that comes with the sdf model would be ignorable, just a default. A pose specified while reading in the robot should override, I think. So: use sdf pose if none given, otherwise override rather than consider them in conflict.
- Isn't there a standard "pose" data structure we can use here and everywhere a pose is required in Drake? The separate xyz,rpy parameters seem awkward to me, especially since they both use the identical Vector3d type and could be inadvertently switched (not to mention that only one of them is actually a vector!). I believe Eigen provides a collection of good pose representations -- can we standardize on one of them?